onadata/apps/logger/migrations/0061_auto_20200713_0814.py

```diff
@@ -0,0 +1,26 @@
+# pylint: skip-file
+# Generated by Django 2.2.10 on 2020-07-13 12:14
+
+from django.db import migrations
+from onadata.libs.utils.common_tools import get_uuid
+
+
+def generate_uuid_if_missing(apps, schema_editor):
+    """
+    Generate uuids for XForms without them
+    """
+    XForm = apps.get_model('logger', 'XForm')
+
+    for form in XForm.objects.filter(uuid=''):
```

Comment:
We tend to use `xform` instead of form as a variable of an `XForm` model object. Can we keep the same for consistency?

Reply:
Updated this to `xform` in the latest commits